frb_example/pure_dart_multi/rust/Cargo.toml

```diff
@@ -8,6 +8,7 @@ name = "flutter_rust_bridge_example_multi"
 crate-type = ["cdylib"]
 
 [features]
+no-wasm = []
```

Comment:
nit: I wonder why do we add no-wasm flag? multi-block seems universal

Reply:
In [2b994b4](https://github.com/fzyzcjy/flutter_rust_bridge/pull/1145/commits/2b994b426b04a049b2539c88b61e5a9dfa31db64), I refined it, since the `features` is almost no needed after erasing those `build.rs` script files.  

Correspondingly, the tests in file justfile are also compacted.  Note: I also flipped the order of  `just rust_build_and_test` and `just generate_all` under block `just precommit`, because within the original order, if there is no generation work done before, task `rust_build_and_test` would fail since there are no essential files---This vulnerability is somewhat covered up by the test module under the unit tests in `frb_codegen\src\main.rs` since usually the tests(files would be generated)would be run before `just precommit`.